{"name": "test_valid_credentials[chrome]", "status": "failed", "statusDetails": {"message": "AssertionError: assert 'Account Login' == 'My Account'\n  \n  - My Account\n  + Account Login", "trace": "self = <QATests.tests.test_login.TestLogin object at 0x000002A596EEBE00>\n\n    def test_valid_credentials(self):\n        login_page=LoginPage(self.driver)\n        login_page.set_email_address(TestData.email)\n        login_page.set_password(TestData.password)\n        login_page.click_login_button()\n        actual_title=login_page.get_title()\n>       assert actual_title == \"My Account\"\nE       AssertionError: assert 'Account Login' == 'My Account'\nE         \nE         - My Account\nE         + Account Login\n\ntests\\test_login.py:14: AssertionError"}, "attachments": [{"name": "stdout", "source": "538f5c4c-dfdc-40ac-b010-5ad3f3e8b92c-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "initialize_driver", "value": "'chrome'"}], "start": 1742851544701, "stop": 1742851546068, "uuid": "e3fe9acf-b044-4b88-b845-94102b104945", "historyId": "2419c62d5ca8d99ec590e5035b591637", "testCaseId": "a0e59e5414dc0cdb0a8124e2ef835da1", "fullName": "QATests.tests.test_login.TestLogin#test_valid_credentials", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('initialize_driver')"}, {"name": "parentSuite", "value": "QATests.tests"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "ShawnAcer"}, {"name": "thread", "value": "80872-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "QATests.tests.test_login"}]}